nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1140,7 +1141,29 @@ define([
                 var tab = message.data.tab;
                 toggleTab(tab);
             });
-            return buttons;
+
+            var outdatedBtn = a({
```

Comment:
This code adds the button to the app control panel, but it doesn't get un-hidden in this version. Easy enough to swap out, if that's what we settle on.